features/shop/checkout/paying_for_order/preventing_payment_step_completion_without_a_selected_method.feature

```diff
@@ -10,7 +10,7 @@ Feature: Preventing payment step completion without a selected method
         And the store has a product "PHP T-Shirt" priced at "$19.99"
         And I am a logged in customer
 
-    @todo @ui @todo
+    @no-api @ui @javascript
```

Comment:
I guess that this scenario and below ones could make sense in API context, so add here `@todo-api` tag 🤔